mesop/examples/__init__.py

```diff
@@ -1,4 +1,5 @@
 from demo import main as main
+from demo import markdown_editor as markdown_editor
```

Comment:
you don't need to import it here

Reply:
Instead import it in https://github.com/google/mesop/blob/d132af54d52d7155fca569aa48ec64c3f2968e81/demo/main.py#L22

And then add it as an Example inside one of the Section, e.g. https://github.com/google/mesop/blob/d132af54d52d7155fca569aa48ec64c3f2968e81/demo/main.py#L77 and then it'll show up in the side menu. The embed should work then.

Reply:
Thanks. That worked.

A few side notes:

- For the markdown editor, the UI gets a bit squished in the demo preview view on a laptop. Could be nice to have a way to hide the code view. Or do a "full screen" embed view.
- The hot reloading doesn't seem to track the changes in demo folder. I'm forced to stop and start the CLI. I think execute_module.py would need updates to handle it. Although I think maybe you have related hot reloading bug already in Github for this. Haven't checked.